runtime/src/bank.rs

```diff
@@ -638,6 +646,21 @@ impl Bank {
         self.slots_per_epoch
     }
 
+    /// Return the number of slots until the next leader schedule activates.
+    pub fn leader_schedule_offset(&self) -> u64 {
```

Comment:
`leader_schedule_offset()` is confusing to me because it contains no units or reference point (ie, offset from what?).  How about something like `leader_schedule_remaining_slots()`?

Reply:
Ok, let me think on that for a sec. I might even be able to delete it from the public interface.

Reply:
Ok, pushed a new commit. better?  It's not "remaining slots".  I don't have a clean way to describe it, but the book calls it "leader schedule offset".